util/MySqlMigrations/Migrations/DatabaseContextModelSnapshot.cs

```diff
@@ -2676,8 +2678,12 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
             modelBuilder.Entity("Bit.Infrastructure.EntityFramework.SecretsManager.Models.ServiceAccount", b =>
                 {
+                    b.Navigation("ApiKeys");
+
                     b.Navigation("GroupAccessPolicies");
 
+                    b.Navigation("ProjectAccessPolicies");
+
                     b.Navigation("UserAccessPolicies");
                 });
```

Comment:
@rkac-bw this is unrelated to my change, but this is an autogenerated file, and this doesn't appear in the migration itself. Is this OK?

Reply:
From what I understand, this file is generated and updated by EF when a migration is added or applied It is  a point-in-time reference of the model to assist in migrations. It should be fine.  @withinfocus Thoughts?